apple/Sources/FerrostarMapLibreUI/Views/LandscapeNavigationView.swift

```diff
@@ -8,9 +8,11 @@ import SwiftUI
 
 /// A landscape orientation navigation view that includes the InstructionsView and ArrivalView on the
 /// leading half of the screen.
-public struct LandscapeNavigationView: View {
```

Comment:
I assume `CustomizableNavigatingInnerGridView` was omitted by accident here - it's present in DynamicallyOrienting and Portrait NavigationView's.